shex/sio-model/personShape.shex

```diff
@@ -0,0 +1,18 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v010/shex/>
```

Comment:
Hi @rajaram5, is the version `/v010/` correct?

Reply:
Nice catch, I will fix version in the IRI